### ISSUE TITLE
[Bug]:[restful v2] If the consistencyLevel value is not set to a valid value, the request will not return an error but instead return an empty string.

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#36444]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:ma
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

![image](https://github.com/user-attachments/assets/45c397f8-2057-4a1c-9ba9-69b5ce51938d)
```
        payload = {
            "collectionName": name,
            "data": [vector_to_search],
            "outputFields": output_fields,
            "limit": limit,
            "offset": 0,
            "consistencyLevel": "Eventual"
        }
```

### Expected Behavior

Displayed error messages, indicating the cause of the error.

**Comment by zhuwenxing** [comment]
The optional enumeration values are Strong, Bounded, Eventually, Session, and Customized. If other values are set, the response will not return an error, but no value will be returned.

**Comment by smellthemoon** [comment]
pr merged. could you please help to verify? @zhuwenxing

**Comment by zhuwenxing** [comment]
```
[2024-11-05 16:05:33 - DEBUG - ci_test]: 
method: post, 
url: http://10.104.30.21:19530/v2/vectordb/collections/create, 
cost time: 0.0573267936706543, 
header: {'Content-Type': 'application/json', 'Authorization': 'Bearer None', 'RequestId': 'ba9a1094-9b4c-11ef-83bc-acde48001122'}, 
payload: {
    "collectionName": "test_collection_2024_11_05_16_05_32_340038OEBESkud",
    "dimension": 128,
    "metricType": "COSINE",
    "params": {
        "enableDynamicField": false,
        "shardsNum": 2,
        "consistencyLevel": "Strong1",
        "ttlSeconds": 360
    }
}, 
response: {"code":1100,"message":"consistencyLevel can only be [Strong, Session, Bounded, Eventually, Customized], default: Bounded: invalid parameter[expected=Strong, Session, Bounded, Eventually, Customized][actual=Strong1]"} (milvus.py:81)
FAILED
```

verified and fixed in `master-20241105-b83b376c-amd64`
